Xamarin.Forms.Core/IndicatorView.cs

```diff
@@ -17,7 +17,8 @@ public class IndicatorView : TemplatedView
 		public static readonly BindableProperty CountProperty = BindableProperty.Create(nameof(Count), typeof(int), typeof(IndicatorView), default(int), propertyChanged: (bindable, oldValue, newValue)
 			=> (((IndicatorView)bindable).IndicatorLayout as IndicatorStackLayout)?.ResetIndicatorCount((int)oldValue));
 
-		public static readonly BindableProperty MaximumVisibleProperty = BindableProperty.Create(nameof(MaximumVisible), typeof(int), typeof(IndicatorView), int.MaxValue);
+		public static readonly BindableProperty MaximumVisibleProperty = BindableProperty.Create(nameof(MaximumVisible), typeof(int), typeof(IndicatorView), int.MaxValue, propertyChanged: (bindable, oldValue, newValue)
+		=> (((IndicatorView) bindable).IndicatorLayout as IndicatorStackLayout)?.ResetIndicators());
```

Comment:
specializing the generic code for a single special case of IndicatorLayout doesn't look like good design